### PR TITLE
Complete library application

### PR DESCRIPTION
**Pull request by canriquez**
Hi, please review the following project.
In this PR we include the full code application as requested in the odin project.

### CODE REVIEW

File: assets/js/main.js [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+/* eslint-enable */
+
+let book = new Book('Atomic Habits', 'James Clear', true, 'Build habits and change your life');
+myLibrary.push(book);
+book = new Book('In Search of Lost Time ', 'Marcel Proust', true, "Swann's Way, the first part of A la recherche de temps perdu");
+myLibrary.push(book);
+book = new Book('Ulysses ', 'James Joyce', false, 'Ulysses chronicles the passage of Leopold Bloom through Dublin during an ordinary day, June 16, 1904.');
+myLibrary.push(book);
+book = new Book(' Don Quixote', 'Miguel de Cervantes', false, 'Alonso Quixano, a retired country gentleman in his fifties,');
+myLibrary.push(book);
```
Comment (Oghenebrume50) [inline assets/js/main.js:131]: improve the save book function to take parameters so you do not have to create a new book on one line and call mylibrary.push again

File: assets/js/main.js [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+    }
+}
+/* eslint-enable */
+
+/* eslint-disable */
+function removeBook(index) {
+    myLibrary.splice(index, 1);
+    refreshList();
+}
+/* eslint-enable */
```
Comment (Oghenebrume50) [inline assets/js/main.js:122]: there are a bunch of these everywhere, is there a particular reason why? :thinking:

File: assets/js/main.js (hunk 1 above)
Comment (canriquez) [reply assets/js/main.js:131]: this has been refactored as requested.

File: assets/js/main.js (hunk 2 above)
Comment (canriquez) [reply assets/js/main.js]: We disabled eslint as it was reacting to functions that were declared but not used. these functions were originally called from HTML "onclick" events.

Comment (canriquez) [reply assets/js/main.js]: Now we have refactored this methods and all functions are called within the main.js. As a result, now we only disable eslint one time (as it is not possible to comply with the eslint suggested rule as we have a circular dependency on functions).

**Comment by canriquez** [comment]
> # Approved ✔️
> ### Hello team 🌟
> ### _Nice work on this project, it seems that you have met the requirements. 👍_
> ### You may merge to master and submit a project completion form. 🚀
> ### Good luck! 🍀

Thanks for the help Radouane! It is so great to have a familiar helping us in this project!.

**Pull request merged**